README.md

```diff
@@ -17,6 +17,8 @@ p = PageviewsClient(user_agent="<person@organization.org> Selfie, Cat, and Dog a
 p.article_views('en.wikipedia', ['Selfie', 'Cat', 'Dog'])
 p.project_views(['ro.wikipedia', 'de.wikipedia', 'commons.wikimedia'])
 p.top_articles('en.wikipedia', limit=10)
+p.top_articles_by_country('en.wikipedia', limit=10)
+p.top_articles_per_country('in', limit=10)
```

Comment:
I leave this up to you, if you want to make it easier for search/replace later, you can make it `top_per_country` but I agree that `top_articles_per_country` is a much better name.

Reply:
agreed -- no perfect name and this aligns with the REST API naming so I switched it to `top_per_country`